cfdtools/hdf5/_hdf5.py

```diff
@@ -30,8 +30,7 @@ def open(self, mode='r', datatype=None):
         try:
             self._h5file = h5py.File(self._path, mode=mode)
         except NameError:
-            io.print('error', "h5py could not be imported")
-            raise
+            raise NameError(f'{self._path} can not be open in {mode} mode.')
```

Comment:
18 files use f-string `f"..."`
2 files use f-string `f'...'` (this one and cfdtools/api.py and cfdtools/cgns/cgns.py)
Standardize ? (except when a literal `'"'`is needed)